apps/api/src/validation/dynamic-data-type-finder.service.ts

```diff
@@ -0,0 +1,27 @@
+import { bootstrapDynamicTypes } from '@coscrad/data-types';
+import { DiscoveryService } from '@golevelup/nestjs-discovery';
+import { Injectable } from '@nestjs/common';
+import { Ctor } from '../lib/types/Ctor';
+
+const isClass = (input): input is Ctor<unknown> => {
+    return typeof input === 'function' && /^\s*class\s+/.test(input.toString());
```

Comment:
Interesting

Reply:
Yea. Lifted this off stack overflow. Not sure if I ended up needing it after I changed the way I introduce data classes into the IoC containers.